cli.go

```diff
@@ -368,20 +358,35 @@ func printResult(out io.Writer, result *Result, mode DisplayMode, withStats bool
 	}
 }
 
-// buildDdlStatements build batched statement only if all statements are DDL statements
-func buildDdlStatements(input string) Statement {
-	var ddls []string
+func buildStatementsWithFlag(input string) ([]*statementWithFlag, error) {
+	var stmts []*statementWithFlag
+	var pendingDdls []string
 	for _, separated := range separateInput(input) {
 		stmt, err := BuildStatement(separated.statement)
 		if err != nil {
-			return nil
+			return nil, err
 		}
-
 		if ddl, ok := stmt.(*DdlStatement); ok {
-			ddls = append(ddls, ddl.Ddl)
-		} else {
-			return nil
+			pendingDdls = append(pendingDdls, ddl.Ddl)
+			continue
 		}
+
+		// Flush pending DDLs
+		stmts = append(stmts, buildStatementWithFlagFromDdls(pendingDdls)...)
```

Comment:
I think it's more readable to check if `pendingDdls` contains element.
```
if len(pendingDdls) > 0 {
  stmts = append(stmts, &statementWithFlag{&DdlStatements{ddls}, false})
  pendingDdls = nil
}
```
Then, we don't need to use `buildStatementWithFlagFromDdls` function.

Reply:
Also could you consider renaming `DdlStatements` to `BulkDdlStatement` so that every statement has consistent suffix `Statement`.

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/40/commits/967f21575dcd6c415753ec50aaadc6afff125c0c and https://github.com/cloudspannerecosystem/spanner-cli/pull/40/commits/cb8db1cc80f8f854e9da1d1fe2fa500cfc473e78